crates/primitives/src/constants/mod.rs

```diff
@@ -92,6 +92,16 @@ pub const HOLESKY_GENESIS_HASH: B256 =
 pub const DEV_GENESIS_HASH: B256 =
     b256!("2f980576711e3617a5e4d83dd539548ec0f7792007d505a3d2e9674833af2d7c");
 
+/// Optimism goerli genesis hash.
+#[cfg(feature = "optimism")]
+pub const GOERLI_OP_GENESIS: B256 =
+    b256!("c1fc15cd51159b1f1e5cbc4b82e85c1447ddfa33c52cf1d98d14fba0d6354be1");
+
+/// Base goerli genesis hash.
+#[cfg(feature = "optimism")]
```

Comment:
we can keep these without cfg

Reply:
addressed in https://github.com/anton-rs/op-reth/pull/135